filament/backend/src/opengl/ShaderCompilerService.cpp

```diff
@@ -772,26 +774,25 @@ UTILS_NOINLINE
         } else {
             line = shader.substr(start, end - start);
         }
-        out << lc++ << ":   " << line.c_str() << '\n';
+        LOG(ERROR) << lc++ << ":   " << line.c_str();
```

Comment:
I wonder if this should be a utils/std::stream, and then output to LOG(ERROR) at the end.

Reply:
Yeah, I thought about that. The difference is that with `LOG(ERROR)`, Abseil prepends metadata for each line. I guess we have to decide which style we prefer.